packages/react-devtools-extensions/src/astUtils.js

```diff
@@ -36,11 +36,7 @@ function checkNodeLocation(
 ): boolean {
   const {start, end} = path.node.loc;
 
-  if (line < start.line || line > end.line) {
-    return false;
-  }
-
-  if (line !== start.line) {
+  if (line !== start.line || line > end.line) {
```

Comment:
I don't understand why `line !== start.line` isn't sufficient. Can you explain the case I'm overlooking?

Reply:
discussed offline, since we're being more strict about checking for the start line, it doesn't make sense to still try to check if the line is within range of the end line, so we can remove that check altogether